src/SALib/sample/morris/morris.py

```diff
@@ -85,28 +77,28 @@ def sample(problem: dict, N: int, num_levels: int = 4,
 
     Returns
     -------
-    sample : numpy.ndarray
+    sample_morris : numpy.ndarray
         Returns a numpy.ndarray containing the model inputs required for Method
         of Morris. The resulting matrix has :math:`(G/D+1)*N/T` rows and
         :math:`D` columns, where :math:`D` is the number of parameters.
     """
     if seed:
         np.random.seed(seed)
 
-    check_if_num_levels_is_even(num_levels)
+    _check_if_num_levels_is_even(num_levels)
```

Comment:
typing?

Reply:
This is actually a function call, not a definition. There is a `num_levels: int` in the function definition.

Reply:
Oops, you are correct!